src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,11 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoryList = createSlice({
+  name: 'categories',
+  initialState: {
+    categories: [],
+    status: 'Under construction',
+  },
+});
```

Comment:
- [ ] Here we have to 
- Creates an initialState with a state value categories that contains an empty array. (This is done)
- Creates 1 reducer, checkStatus
We have to create a reducer separately. For ex:-
```javascript
const categoriesSlice = createSlice({
  name: "categories",
  initialState: {
    categories: [],
  },
  reducers: {
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
  },
})
```